public/v4/apps/ombi.yml

```diff
@@ -0,0 +1,35 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        image: linuxserver/ombi:$$cap_ombi_version
+        environment:
+            TZ: $$cap_tz
+            PUID: '1000'
+            PGID: '1000'
+        restart: unless-stopped
+        volumes:
+            - $$cap_appname-config:/config
+        caproverExtra:
+            containerHttpPort: '3579'
+caproverOneClickApp:
+    variables:
+        - id: $$cap_ombi_version
+          label: Ombi Version
+          defaultValue: development-v4.0.1446-ls143
```

Comment:
Usually it's better to go with stable versions, like [version-v4.0.497](https://hub.docker.com/layers/linuxserver/ombi/version-v4.0.497/images/sha256-97e801a46a4ddec5ddbdc2ce4f07502e00e60ce6e9e275a0868628344f597106?context=explore)

Reply:
This one doesn't have a stable version for now.

Reply:
`version-v4.0.497` isn't stable?

Reply:
Yes, it is. I didn't found that one. lol